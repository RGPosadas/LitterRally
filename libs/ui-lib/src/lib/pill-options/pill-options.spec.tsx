import React from 'react';
import { render } from '@testing-library/react';

import RestrictionOptions from './restriction-options';

describe('RestrictionOptions', () => {
  it('should render successfully', () => {
    const { baseElement } = render(<RestrictionOptions />);
    expect(baseElement).toBeTruthy();
  });
});
