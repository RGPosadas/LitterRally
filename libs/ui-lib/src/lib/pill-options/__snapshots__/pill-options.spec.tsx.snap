// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PillOptions should match snapshot 1`] = `
<div
  className="pill-options"
>
  <div
    className="pill-option active"
    data-testid="pill-id-0"
    onClick={[Function]}
  >
    Yes
  </div>
  <div
    className="pill-option "
    data-testid="pill-id-1"
    onClick={[Function]}
  >
    No
  </div>
  <div
    className="pill-option "
    data-testid="pill-id-2"
    onClick={[Function]}
  >
    Maybe
  </div>
</div>
`;
